modules/idxBidAdapter.js

```diff
@@ -0,0 +1,80 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js'
+import { BANNER } from '../src/mediaTypes.js'
+import { isArray, isNumber } from '../src/utils.js'
+
+const BIDDER_CODE = 'idx_bidder_code'
```

Comment:
Can we also shorten the `BIDDER_CODE` value to simply `idx`? This would make the configuration more readable. For example: 
```
    var adUnits = [{
      code: 'your-slot-div-id', 
      mediaTypes: {
          banner: {
              sizes: [[300, 250]]
          }
      },
      bids: [{
          bidder: 'idx',
          params: {}
      }]
    }]
```